src/tests/ftest/pool/create_all.py

```diff
@@ -0,0 +1,192 @@
+        self.assertLessEqual(abs(smd_avail_bytes-2*tier_bytes[1]), epsilon_bytes,
+                f"Invalid SMD size: want={smd_avail_bytes/2}, got={tier_bytes[1]}, epsilon={epsilon_bytes}")
+
+        # NOTE The size of the available bytes is different from he size of the first pool: size
+        # used by the metadata are removed from the available size returned.
+        self.log.info("Checking size of available storage after the creation of the first pool")
+        tier_bytes = self.get_available_bytes()
+        self.assertLessEqual(abs(scm_avail_bytes-2*tier_bytes[0]), max_metadata_bytes,
+                f"Invalid SCM size: want={scm_avail_bytes/2}, got={tier_bytes[0]}")
+        self.assertLessEqual(abs(smd_avail_bytes-2*tier_bytes[1]), max_metadata_bytes,
+                f"Invalid SMD size: want={smd_avail_bytes/2}, got={tier_bytes[1]}")
+        scm_avail_bytes, smd_avail_bytes = tier_bytes
+
+        self.log.info("Creating a second pool with 100% of the available storage")
+        self.pool[1].create()
+        self.pool[1].get_info()
+        self.assertEqual(self.pool[1].dmg.result.exit_status, 0,
+                "Second pool could not be created")
+
+        self.log.info("Checking size of the second pool")
+        self.pool[1].get_info()
+        tier_bytes = self.pool[1].info.pi_space.ps_space.s_total
+        self.assertLessEqual(abs(scm_avail_bytes-tier_bytes[0]), epsilon_bytes,
+                f"Invalid SCM size: want={scm_avail_bytes}, got={tier_bytes[0]}, epsilon={epsilon_bytes}")
+        self.assertLessEqual(abs(smd_avail_bytes-tier_bytes[1]), epsilon_bytes,
+                f"Invalid SMD size: want={smd_avail_bytes/2}, got={tier_bytes[1]}, epsilon={epsilon_bytes}")
+
+        self.log.info("Checking size of available storage after the creation of the second pool")
+        tier_bytes = self.get_available_bytes()
+        self.assertEqual(0, tier_bytes[0],
+                f"Invalid SCM size: want=0, got={tier_bytes[0]}")
+        self.assertEqual(0, tier_bytes[1],
+                f"Invalid SMD size: want=0, got={tier_bytes[1]}")
+
+    def test_recycle_pools(self):
+        """Test pool creation and destruction
+
+        Test Description:
+            Create a pool with all the capacity of all servers. Verify that the pool created
+            effectively used all the available storage. Destroy the pool and repeat these steps 100
+            times. For each iteration, check that the size of the created pool is always the same.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,large
+        :avocado: tags=pool
+        :avocado: tags=pool_create_tests,pool_create_all,pool_create_all_recycle
+        """
+
+        scm_delta_bytes = 1 << 20 # 1MiB
+        scm_avail_bytes, smd_avail_bytes = self.get_available_bytes()
+
+        for index in range(10):
+            self.log.info(f"Creating pool {index} with 100% of the available storage")
```

Comment:
(pylint-logging-fstring-interpolation) Use lazy % formatting in logging functions